drivers/net/slip.c

```diff
@@ -1,4 +1,4 @@
-/****************************************************************************
+/******************************************************************************
```

Comment:
should we keep the max line size(79)?

Reply:
Ok, will do.

For reference: [NuttX C Coding Standard - Lines](https://nuttx.apache.org/docs/latest/contributing/coding_style.html#lines)